acceptance/setup/git/pre-suite/030_PuppetMasterSanity.rb

```diff
@@ -1,6 +1,6 @@
 test_name "Puppet Master sanity checks: PID file and SSL dir creation"
 
-pidfile = '/var/lib/puppet/run/master.pid'
+pidfile = '/var/run/puppetlabs/agent/master.pid'
```

Comment:
Ah right. So we could change `rundir` to `/var/run/puppetlabs` and just set the default path for the pidfile to `$rundir/${run_mode}/${run_mode}.pid`? We'd have to also be sure that install.rb creates both `/var/run/puppetlabs/agent` and `/var/run/puppetlabs/master`, and that AIO packaging updates reflect this.

Thoughts?